distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,111 @@
+
+    // Change current security manager
+    permission java.lang.RuntimePermission "setSecurityManager";
+
+    // Modify application permissions at will.
+    permission java.security.SecurityPermission "setPolicy";
+
+    // Load classes into any protection domain
+    permission java.lang.RuntimePermission "createClassLoader";
+};
+
+//
+// DDF Libraries have broad file permissions within the install directory
+//
+grant
+  codeBase "file:${ddf.home.policy}system/ddf/-" {
+    permission java.io.FilePermission "${ddf.home}", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}-", "read, write";
+
+    permission java.util.PropertyPermission "javax.net.ssl.*", "read, write";
+};
+grant
+  codeBase "file:${ddf.home.policy}system/org/codice/-" {
+    permission java.io.FilePermission "${ddf.home}", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}-", "read, write";
+
+    permission java.util.PropertyPermission "javax.net.ssl.*", "read, write";
+};
+
+//
+// The lib directory contains root, trusted jars for DDF, OSGi, and security
+//
+grant
+  codeBase "file:${ddf.home.policy}lib/-" {
+    permission java.io.FilePermission "${ddf.home}", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}*", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}-", "read";
+
+    permission java.lang.RuntimePermission "createClassLoader";
+    permission java.util.PropertyPermission "javax.net.ssl.*", "read, write";
+};
+grant
+  codeBase "file:${ddf.home.policy}lib/boot/-" {
+    permission java.io.FilePermission "${ddf.home}${/}*", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}config.properties", "write";
+};
+grant
+  codeBase "file:${ddf.home.policy}lib/boot/org.osgi.core-6.0.0.jar" {
+    // Needed to load fonts
+    permission java.io.FilePermission "<<ALL FILES>>", "read, execute";
+};
+
+//
+// Core OSGi libraries
+//
+grant
+  codeBase "file:${ddf.home.policy}system/org/eclipse/platform/org.eclipse.osgi/-" {
+    // Needed to load fonts
+    permission java.io.FilePermission "<<ALL FILES>>", "read, execute";
+
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}-", "read, write";
+
+    permission java.lang.RuntimePermission "createClassLoader";
+    permission java.util.PropertyPermission "javax.net.ssl.*", "read";
+};
+
+grant {
+    // User's home directory
+    permission java.io.FilePermission "${user.home}${/}-", "read, write";
+
+    // Java libraries
+    permission java.io.FilePermission "${java.home}${/}release", "read";
+    permission java.io.FilePermission "${java.home}${/}lib${/}-", "read";
+
+    // System libraries
+    permission java.io.FilePermission "/dev/urandom", "read";
+    permission java.io.FilePermission "/lib", "read";
+    permission java.io.FilePermission "/proc/self/exe", "read";
+    permission java.io.FilePermission "/usr/lib", "read";
+
```

Comment:
❓ How about temporary folders, e.g., `/tmp`, `/var/tmp`, `C:\TEMP`, etc.? The `FileBackedOutputStream` class (which is used in quite a few places in our code) needs to be able to create temporary files: https://github.com/google/guava/blob/master/guava/src/com/google/common/io/FileBackedOutputStream.java#L195.

Reply:
We will need to test this across platforms, but it is almost definitely broken right now.

Reply:
Even though things worked because `FileBackedOutStream` defaults to a directory under `ddf.home`, we may still want to grant read access to the `java.io.tmpdir` system property and allow read/write access to the directory it points to. This would prevent any unexpected issues if a third-party library decides to create some temporary files there.

Reply:
don't forget delete too

Reply:
Thanks, @paouelle. Fixed.